components/content/TableOfContents.jsx

```diff
@@ -5,7 +5,7 @@ import { FaTimes } from "react-icons/fa";
 
 const TableOfContents = ({ toggleMenu }) => {
   return (
-    <div role="navigation">
```

Comment:
I would suggest rather than deleting the role, change the `<div>` to a `<nav>`. A table of content is kind of a navigation, this information is useful for screen reader users. If the accessibility test still fails, add an aria-label with table of contents or such, to make the `<nav>` unique. This should do the trick. If not, please reach out to me 😄